src/vs/workbench/contrib/welcomeGettingStarted/browser/featuredExtensionService.ts

```diff
@@ -86,12 +86,12 @@ export class FeaturedExtensionsService extends Disposable implements IFeaturedEx
 
 		const extensions = await Promise.race([
 			this.tasExperimentService?.getTreatment<string>('welcome.featured.item'),
-			new Promise<string | undefined>(resolve => setTimeout(() => resolve(''), 2000))
+			new Promise<string | undefined>(resolve => setTimeout(() => resolve(''), 100))
```

Comment:
Since latency is critical for the welcome page (essentially adding to start up time), I still think we should just cache this value for the next time the welcome page starts up. And just use the stale value for the current load. Even adding 100-200ms is bad.

Alternatively, allow the extensions to load in after the welcome page has started up, so they fill in over the next second or something. This isn't so bad if it's just the extensions part filling in.

Reply:
That was the previous behavior where the extensions were filling in after reload. I can change it back to the previous behavior like below.
![Recording 2023-06-08 at 08 46 57](https://github.com/microsoft/vscode/assets/25044782/50110751-36a0-40a7-a369-01f0332285dc)

Reply:
How about we do that but move the recommended extension to below the walkthroughs so the page doesn't shift around?

Reply:
@digitarald what do you think?

Concern was that with a long list of walkthroughs the extension would render all the way down and would require scrolling to see it. 
Also we have an ongoing experiment.  Which would skew the data.

Reply:
Since the experiment is wrapped up we can move it down for now and fix the layout shift later.

Reply:
The behavior shown in https://github.com/microsoft/vscode/pull/184573#discussion_r1223267059 looks reasonable to me, it ensures an instant load of welcome page and the network or remote extension connection is only affecting the load of the recommendation card.

Maybe a bit of `transition` animation can make the appear of the card and the shift of walkthroughs look nice, which is probably better than putting the recommendation below the walkthroughs IMHO.

Reply:
Updated my changes to mostly revert changes to the previous experience or delete dependence on experiment service. 
I'm not comfortable changing up the experience to show the walkthroughs above the extension or adding transitions for this change.

We can explore improving the experience later.